components/meter/meter-linear.js

```diff
@@ -0,0 +1,53 @@
+import { html, LitElement } from 'lit-element/lit-element.js';
+import { meterStyles } from './meter-styles.js';
+import { RtlMixin } from '../../mixins/rtl-mixin.js';
+
+class MeterLinear extends RtlMixin(LitElement)  {
+	static get styles() {
+		return [ meterStyles ];
+	}
+
+	render() {
```

Comment:
Nit: could the `_computeProgressBarWidth` calculation just be done right here? Like:

```
const progressBarWidth = this.units / this.outOf * this._actualWidth;
```

Seems unnecessary to put it in a method.